lib/components/viewers/nearby/styled.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components'
+
+export const FloatingLoadingIndicator = styled.div`
+  aspect-ratio: 1;
+  background: white;
+  border-radius: 1rem;
+  color: black;
+  left: 0px;
+  margin: 5px;
+  padding: 3px;
+  position: fixed;
+`
+
+export const NearbySidebarContainer = styled.ol`
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+  padding: 0 3rem;
```

Comment:
Use 1em for horizontal padding to be consistent with the search results.

Reply:
Oops, I did this but forgot to push it.